### PR TITLE
Add LBNL to acknowledgment

### LINKED ISSUE #161: Update LBNL acknowledgement
Can we please ask a favor? We would like to add two sentences to the acknowledgments listed on https://www.legacysurvey.org/acknowledgment/ under “Scientific Publication Acknowledgment”: 

- In the first para, please add the following as the penultimate sentence: 
	"Pipeline processing and analyses of the data were supported by NOIRLab and the Lawrence Berkeley National Laboratory. “

- In the second para, please add the following: 
	"LBNL is managed by the Regents of the University of California under contract to the U.S. Department of Energy. “

So now, the first two paras should read as follows (changes highlighted): 
The Legacy Surveys consist of three individual and complementary projects: the Dark Energy Camera Legacy Survey (DECaLS; Proposal ID #2014B-0404; PIs: David Schlegel and Arjun Dey), the Beijing-Arizona Sky Survey (BASS; NOAO Prop. ID #2015A-0801; PIs: Zhou Xu and Xiaohui Fan), and the Mayall z-band Legacy Survey (MzLS; Prop. ID #2016A-0453; PI: Arjun Dey). DECaLS, BASS and MzLS together include data obtained, respectively, at the Blanco telescope, Cerro Tololo Inter-American Observatory, NSF’s NOIRLab; the Bok telescope, Steward Observatory, University of Arizona; and the Mayall telescope, Kitt Peak National Observatory, NOIRLab. *Pipeline processing and analyses of the data were supported by NOIRLab and the Lawrence Berkeley National Laboratory (LBNL).* The Legacy Surveys project is honored to be permitted to conduct astronomical research on Iolkam Du’ag (Kitt Peak), a mountain with particular significance to the Tohono O’odham Nation.

NOIRLab is operated by the Association of Universities for Research in Astronomy (AURA) under a cooperative agreement with the National Science Foundation. *LBNL is managed by the Regents of the University of California under contract to the U.S. Department of Energy.*

### PR DESCRIPTION
**Pull request by weaverba137**
This PR closes #161.

### CODE REVIEW

**Comment by geordie666** [comment]
@weaverba137: This looks correct to me. But, I wonder if we want to address #157 while we're updating the acknowledgments section? It's your call — also feel free to merge this now and I can deal with #157 in a later PR.

**Review by geordie666 — APPROVED** [review]

**Comment by weaverba137** [comment]
I hadn't seen that issue, but I will definitely add that to this PR.

**Comment by weaverba137** [comment]
@geordie666, short acknowledgment is incorporated. Does the short acknowledgment also need to explicitly mention LBNL, or is that automatically included in the standard DESI acknowledgments?

File: pages/acknowledgment.rst [hunk 1]
```diff
@@ -103,6 +103,30 @@ by the National Energy Research Scientific Computing Center, a DOE Office of Sci
...
+under a cooperative agreement with the National Science Foundation.
+Legacy Surveys also uses data products from the Near-Earth Object Wide-field Infrared Survey Explorer (NEOWISE),
+a project of the Jet Propulsion Laboratory/California Institute of Technology, funded by the National Aeronautics and Space Administration.
+Legacy Surveys was supported by: the Director, Office of Science,
+Office of High Energy Physics of the U.S. Department of Energy; the National Energy Research Scientific Computing Center,
+a DOE Office of Science User Facility;  the U.S. National Science Foundation, Division of Astronomical Sciences;
+the National Astronomical Observatories of China, the Chinese Academy of Sciences and the Chinese National Natural Science Foundation.
+The complete acknowledgments can be found at https://www.legacysurvey.org/acknowledgment/.
+
+.. _`standard DESI Acknowledgments`: https://desi.lbl.gov/trac/wiki/GuidetoPub/DESIAcknowledgements
```
Comment (geordie666) [inline pages/acknowledgment.rst:128]: Do we want to change this link to point to the internal location of the "standard" acknowledgment, e.g. 

https://desi.lbl.gov/trac/wiki/GuidetoPub/DESIAcknowledgements#StandardAcknowledgements

?

**Review by geordie666 — COMMENTED** [review]
The language looks correct to me. I have one comment on making the link more specific (which can be dangerous if the internal link is changed).

Decide whether you think it's prudent to update the link, then merge whenever you're ready.

**Comment by geordie666** [comment]
(Oh, and as far as I know we _don't_ need to add the specific LBNL language. The language we're using is what Arjun and David agreed on and sent to the DESI Publications Board).

File: pages/acknowledgment.rst (hunk 1 above)
Comment (weaverba137) [reply pages/acknowledgment.rst:128]: I will add the more specific link and then merge.

**Pull request merged**